src/ecdsa/curves.py

```diff
@@ -469,10 +470,44 @@ def from_pem(cls, string, valid_encodings=None):
 
 
 def find_curve(oid_curve):
+    """Select a curve based on its OID
+
+    :param tuple[int,...] oid_curve: ASN.1 Object Identifier of the
+        curve to return, like ``(1, 2, 840, 10045, 3, 1, 7)`` for ``NIST256p``.
+
+    :raises UnknownCurveError: When the oid doesn't match any of the supported
+        curves
+
+    :rtype: ~ecdsa.curves.Curve
+    """
     for c in curves:
         if c.oid == oid_curve:
             return c
     raise UnknownCurveError(
         "I don't know about the curve with oid %s."
         "I only know about these: %s" % (oid_curve, [c.name for c in curves])
     )
+
+
+def curve_by_name(name):
```

Comment:
As both functions `find_curve()` and `curve_by_name()` do the same (both find a curve), it would be a good idea to unify their function and argument names. Therefore, I would suggest renaming `find_curve(oid_curve)` to `curve_by_oid(oid)`

Reply:
that's changing API, so I'd rather not do that